1-development/user-changes-challenge/server/webApi.js

```diff
@@ -0,0 +1,36 @@
+const service = require('./comparatorService')
+
+/**
+ * /createChanges
+ */
+function createChanges(req, rsp) {
+    service.createChanges(req.body.obj1, req.body.obj2)
+        .then(respObj => {
+            rsp.statusCode = 201
+            rsp.json(respObj)
+        })
+        .catch(errObj => {
+            rsp.statusCode = 400
```

Comment:
Why use a generic 400 error? Is there a better status code that could you use?

Reply:
I used the status code ```400```, which stands for ```The request could not be understood by the server due to malformed syntax. The client should not repeat the request without modifications.```, because the only error that could happen would be bad input from the client. If I had more features that could generate other errors, I would have to check what type of error it was, and then return a different status code.